tests/unittests/test_positioning.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+import sys
+import os
+from pyskindose.phantom_class import Phantom
+import pyskindose.constants as c
+
+test_path = Path(__file__).parent.parent
+sys.path.insert(1, str(test_path.absolute()))
+
+phantom_path = Path(__file__).parent.parent.parent / \
+    'src' / 'pyskindose' / 'phantom_data'
+
+from pyskindose.settings_pyskindose import PyskindoseSettings
+from manual_tests.base_dev_settings import DEVELOPMENT_PARAMETERS
+from pyskindose.geom_calc import position_patient_phantom_on_table
+
+param = PyskindoseSettings(DEVELOPMENT_PARAMETERS)
+
+
+def test_mathematical_phantom_positioning_in_z_direction():
+    # the patient phantom origin is located at the top its head. Therefore,
```

Comment:
Check the spelling/missing words in this sentence

Reply:
fixed in 33a31c900360673925c27eca67cea792c4bd22f4